src/utils/articles.js

```diff
@@ -0,0 +1,106 @@
+import moment from 'moment';
+
+export default {
```

Comment:
@micah-akpan  could you elaborate on why you chose to use export default, and not export.
Also, why are you exporting an object

Reply:
The functions inside this module are all related so I'm putting them here to export a single object as a namespace to all the functions. Do you think named exports would be better in this context?